### DISCUSSION TITLE
MPU-6050 set offset for gyroscope

### DISCUSSION THREAD

**Discussion by geekpaolo** [arendst/Tasmota#15353] [Q&A] [answered]
I need to set offsets for gyroscope.

Where I have to insert the set offset values?

Thanks in advance

**Comment by sfromis** [comment, marked as answer]
https://github.com/arendst/Tasmota/blob/development/tasmota/xsns_32_mpu6050.ino is where it happens, there are some hardcoded offsets when having DMP mode enabled.
